docs/challenge-002.md

```diff
@@ -1,9 +1,47 @@
 # Challenge 2: Invoke a durable entity per store to receive item related messages
 
 ## Challenge Objective
+In Challenge 1 you set up listeners to detect inserts and updates to the onHand and shipments containers.
 
-## Brief Guidance
+In this challenge, we'll send the change feed events to a data manager that maintains its own state and sends them to durable store entities.
```

Comment:
Would something give more context to the challenge objective? 
``` suggestion
In this challenge, we will use the documents updated or added to identify the store and item they related to and message the entity responsible to update the store inventory based on the document received.
```

Reply:
yes, i think that's helpful. thanks!